kornia/geometry/conversions.py

```diff
@@ -55,7 +57,7 @@ class QuaternionCoeffOrder(enum.Enum):
     WXYZ = 'wxyz'
 
 
-def rad2deg(tensor: torch.Tensor) -> torch.Tensor:
+def rad2deg(tensor: Tensor) -> Tensor:
```

Comment:
should we deprecate this one and below? https://pytorch.org/docs/stable/generated/torch.deg2rad.html

Reply:
Not deprecate, but just pass-through to the PyTorch function. I don't like breaking compatibility